<?xml version="1.0" encoding="UTF-8" ?>
<testsuite errors="0" failures="5" hostname="Nicolass-iMac.local" name="racetrack.RunnerControllerTests" tests="8" time="0.611" timestamp="2012-02-22T23:38:22">
  <properties />
  <testcase classname="racetrack.RunnerControllerTests" name="testIndex" time="0.198" />
  <testcase classname="racetrack.RunnerControllerTests" name="testList" time="0.041" />
  <testcase classname="racetrack.RunnerControllerTests" name="testCreate" time="0.04" />
  <testcase classname="racetrack.RunnerControllerTests" name="testSave" time="0.05">
    <failure message="Assertion failed: &#xa;&#xa;assert response.redirectedUrl == &apos;/runner/show/1&apos;&#xa;       |        |             |&#xa;       |        null          false&#xa;       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@684d45c6&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert response.redirectedUrl == &apos;/runner/show/1&apos;
       |        |             |
       |        null          false
       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@684d45c6

	at racetrack.RunnerControllerTests.testSave(RunnerControllerTests.groovy:49)
</failure>
  </testcase>
  <testcase classname="racetrack.RunnerControllerTests" name="testShow" time="0.064">
    <failure message="Assertion failed: &#xa;&#xa;assert runner.save() != null&#xa;       |      |      |&#xa;       |      null   false&#xa;       racetrack.Runner : null&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert runner.save() != null
       |      |      |
       |      null   false
       racetrack.Runner : null

	at racetrack.RunnerControllerTests.testShow(RunnerControllerTests.groovy:64)
</failure>
  </testcase>
  <testcase classname="racetrack.RunnerControllerTests" name="testEdit" time="0.044">
    <failure message="Assertion failed: &#xa;&#xa;assert runner.save() != null&#xa;       |      |      |&#xa;       |      null   false&#xa;       racetrack.Runner : null&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert runner.save() != null
       |      |      |
       |      null   false
       racetrack.Runner : null

	at racetrack.RunnerControllerTests.testEdit(RunnerControllerTests.groovy:83)
</failure>
  </testcase>
  <testcase classname="racetrack.RunnerControllerTests" name="testUpdate" time="0.038">
    <failure message="Assertion failed: &#xa;&#xa;assert runner.save() != null&#xa;       |      |      |&#xa;       |      null   false&#xa;       racetrack.Runner : null&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert runner.save() != null
       |      |      |
       |      null   false
       racetrack.Runner : null

	at racetrack.RunnerControllerTests.testUpdate(RunnerControllerTests.groovy:104)
</failure>
  </testcase>
  <testcase classname="racetrack.RunnerControllerTests" name="testDelete" time="0.068">
    <failure message="Assertion failed: &#xa;&#xa;assert runner.save() != null&#xa;       |      |      |&#xa;       |      null   false&#xa;       racetrack.Runner : null&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert runner.save() != null
       |      |      |
       |      null   false
       racetrack.Runner : null

	at racetrack.RunnerControllerTests.testDelete(RunnerControllerTests.groovy:148)
</failure>
  </testcase>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[--Output from testIndex--
--Output from testList--
--Output from testCreate--
--Output from testSave--
--Output from testShow--
--Output from testEdit--
--Output from testUpdate--
--Output from testDelete--
]]></system-err>
</testsuite>
